go.mod

```diff
@@ -5,8 +5,9 @@ go 1.19
 require (
 	github.com/Azure/aad-pod-identity v1.8.9
 	github.com/Azure/azure-sdk-for-go v67.1.0+incompatible
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.0.0
```

Comment:
Hey @r4f4 -- Can we please use v1.2.0 for azcore?

Reply:
Done